src/utils/servers.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Regex for URL validation
+ */
+export const urlRegEx = /^https?:\/\/.+/;
+
+/**
+ * Parse and returns a list of url strings from DEFAULT_SERVERS env var
+ *
+ * @param {string | undefined} value Env var DEFAULT_SERVERS value
+ * @returns {string[]} list of url strings. (default: empty array)
+ */
+export function parseServerListString(value: string | undefined): string[] {
+  if (!value) return [];
+
+  const serverUrlList: string[] = [];
+
+  /**
+   * Check the URL with the appropriate Regex and if valid, adds it to the list
+   *
+   * @param {string} url Url string to test and add
+   */
+  function checkAndAddUrl(url: string): void {
+    if (urlRegEx.test(url)) {
+      // Valid Url
+      serverUrlList.push(url);
+    } else {
+      console.warn(
+        `[Default Servers] This url is not valid. It will be excluded from the server list to avoid errors. (${url})`
+      );
+    }
+  }
```

Comment:
You can just put that function directly in the foreach below, at least that's what I'd do. @ferferga ?

Reply:
You're right, initially I was using it in two places, that's why I pulled it out.

Now I only use it once, I can actually move it back directly into the forEach

Reply:
It makes sense then :)